agent/config/parse_windows.go

```diff
@@ -16,22 +16,32 @@
 package config
 
 import (
+	"fmt"
 	"os"
 	"strings"
 	"syscall"
 	"unsafe"
 
-	"golang.org/x/sys/windows/registry"
-
 	"github.com/aws/amazon-ecs-agent/agent/utils"
 	"github.com/cihub/seelog"
+	"golang.org/x/sys/windows/registry"
 )
 
 const (
 	// envSkipDomainJoinCheck is an environment setting that can be used to skip
 	// domain join check validation. This is useful for integration and
 	// functional-tests but should not be set for any non-test use-case.
 	envSkipDomainJoinCheck = "ZZZ_SKIP_DOMAIN_JOIN_CHECK_NOT_SUPPORTED_IN_PRODUCTION"
+
+	// The following constants are used to generate Windows OS family types
+	ReleaseId2004SAC          = "2004"
+	ReleaseId1909SAC          = "1909"
+	Windows_Server_2019       = "Windows Server 2019"
```

Comment:
can you remove the underscores and change these to WindowsServer2019 for consistency?